plugins/chat/spec/system/page_objects/chat/chat.rb

```diff
@@ -3,6 +3,12 @@
 module PageObjects
   module Pages
     class Chat < PageObjects::Pages::Base
+      MODIFIER = RUBY_PLATFORM =~ /darwin/i ? :meta : :control
```

Comment:
We have this in multiple places now, we should make some `PageObject::Util::KEY_MODIFIER` const or something that does this:

https://github.com/discourse/discourse/blob/d9321d71b3363434f9be098fafd6aa346e65c882/plugins/chat/spec/system/page_objects/chat/components/composer.rb#L11-L11

https://github.com/discourse/discourse/blob/d9321d71b3363434f9be098fafd6aa346e65c882/plugins/chat/spec/system/chat/composer/shortcuts/channel_spec.rb#L10-L10

Reply:
Yes, I hesitated... not sure what is the best place for it

Reply:
Making a PR to normalize this across the app

Reply:
@jjaffeux https://github.com/discourse/discourse/pull/22451